graylog2-web-interface/src/pages/StreamPermissionErrorPage.jsx

```diff
@@ -0,0 +1,34 @@
+// @flow strict
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router';
+
+import { FetchError } from 'logic/rest/FetchProvider';
+
+import UnauthorizedErrorPage from './UnauthorizedErrorPage';
+
+type Props = {
+  error: FetchError,
+};
+
+const StreamPermissionErrorPage = ({ error }: Props) => {
+  const description = (
+    <>
+      <p>This resource includes streams you do not have permissions to.</p>
+      <p>Please contact your administrator and provide the error details which indlude a list of streams you need access to.</p>
+    </>
+  );
+  const streamIds = error?.additional.body?.streams;
```

Comment:
Is `additional` always present if `error` is, or do we want to make it optional too?

Reply:
Good catch, the `FetchError` should always have `additional`, but we are handling errors with an unknown format e.g. in the `UauthorizedPage` as well, just in case,  so I added the another check.